packages/@icon-magic/config-reader/src/schemas/config-schema.ts

```diff
@@ -41,6 +36,9 @@ const buildConfigProperties = {
 };
 
 const distributeConfigProperties = {
+  filterByVariants: {
```

Comment:
Can we have it both here and under svg if we're now going to contain it only for svg distribute? We can assume that if it's here, it applies to all types of distribution

Reply:
updated.